light-client/src/main/java/tech/pegasys/teku/lightclient/client/Types.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.lightclient.client;
+
+import tech.pegasys.teku.spec.logic.versions.altair.SpecLogicAltair;
+
+public class Types {
```

Comment:
`Types` seems like an overly generic name. We'd normally want the class name to reflect what the actual purpose of the class is and ideally fit into the ubiquitous language (https://thedomaindrivendesign.io/developing-the-ubiquitous-language/).

Reply:
I'll figure out a better name in the future.